programs/util.h

```diff
@@ -118,7 +120,8 @@ U64 UTIL_getFileSize(const char* infilename);
 U64 UTIL_getTotalFileSize(const char* const * fileNamesTable, unsigned nbFiles);
 int UTIL_getFileStat(const char* infilename, stat_t* statbuf);
 int UTIL_setFileStat(const char* filename, stat_t* statbuf);
-int UTIL_chmod(char const* filename, mode_t permissions);   /*< like chmod, but avoid changing permission of /dev/null */
+int UTIL_chmod(char const* filename, mode_t permissions, int onlyRegularFiles);   /*< like chmod, but avoid changing permission of special files */
+int UTIL_chown(char const* filename, uid_t owner, gid_t group, int onlyRegularFiles);   /*< like chown, but avoid changing permission of special files */
```

Comment:
note : if `UTIL_chown()` is only used inside `util.c`, maybe there is no need to publish its interface ?